packages/plugins/Gfi/src/store/getters.ts

```diff
@@ -158,29 +159,19 @@ const getters: PolarGetterTree<GfiState, GfiGetters> = {
   ): Feature[] {
     const { map, clientHeight, clientWidth, center, zoomLevel } = rootGetters
     // trigger getter on those who indicate feature change possibility
-    noop(
-      clientHeight,
-      clientWidth,
-      center,
-      zoomLevel,
-      visibilityChangeIndicator
-    )
+    noop(clientHeight, clientWidth, center, zoomLevel)
+    noop(visibilityChangeIndicator)
```

Comment:
Why in two calls of doing ✨ nothing ✨ ?

Reply:
Doing it in one line makes the auto-linter spread it to seven lines length, breaking the 50-lines-per-function linting rule. I am keeping my poker face up while writing this.